accern_xyme/accern_xyme.py

```diff
@@ -751,6 +753,47 @@ def get_settings(self) -> Dict[str, Any]:
         assert self._settings is not None
         return self._settings
 
+    def get_timing(
+            self,
+            blacklist: List[str] = ["wait_for_uri"],
+            ) -> Dict[str, TimingResult]:
+        nodes = self.get_nodes()
+        pipe_timing: Dict[str, TimingResult] = {}
+        node_timing: Dict[str, NodeTiming] = {}
+
+        def filter_blacklist(
+                node_time: List[Timing]) -> Iterator[Timing]:
+            for value in node_time:
+                if value.get("name") not in blacklist:
```

Comment:
since value is of type Timing you don't need to use get you can just use value["name"]